Dockerfile

```diff
@@ -20,9 +20,6 @@ COPY openedx2zim /src/openedx2zim
 COPY setup.py *.md get_js_deps.sh MANIFEST.in /src/
 RUN cd /src/ && python3 ./setup.py install
 
-COPY entrypoint.sh /usr/local/bin/entrypoint.sh
```

Comment:
what's the reasoning for this? Without this, youtube-dl will never be updated and might fail to download in the future.

Reply:
Got it. Added that again.